style2.css

```diff
@@ -0,0 +1,49 @@
+.burger {
+	background-color: salmon;
+	display: none;
+	height: 3px;
+	position: relative;
+	width: 10px;
+}
+
+.burger:before {
+	content: " ";
+	background-color: salmon;
+	height: 3px;
+	position: absolute;
+	top: 5px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/CSS17K/pullRequest?prid=5818552)